llvm/include/llvm/ADT/EquivalenceClasses.h

```diff
@@ -178,6 +179,9 @@ class EquivalenceClasses {
   member_iterator member_end() const {
     return member_iterator(nullptr);
   }
+  iterator_range<member_iterator> members(iterator I) const {
+    return make_range(member_begin(I), member_end());
+  }
```

Comment:
Could you add a unit test for this to make sure it's exercised outside of the analysis code? I think this would help with the documentation aspect of this code as having it take an iterator is not usual / immediately obvious to me in terms of the semantics.

Reply:
Fixed, thanks.